src/app/games/hive/HiveMinimax.ts

```diff
@@ -0,0 +1,51 @@
+import { BoardValue } from 'src/app/jscaip/BoardValue';
+import { Coord } from 'src/app/jscaip/Coord';
+import { Minimax } from 'src/app/jscaip/Minimax';
+import { Player } from 'src/app/jscaip/Player';
+import { GameStatus } from 'src/app/jscaip/Rules';
+import { MGPOptional } from 'src/app/utils/MGPOptional';
+import { HiveMove } from './HiveMove';
+import { HiveNode, HiveRules } from './HiveRules';
+import { HiveState } from './HiveState';
+
+export class HiveMinimax extends Minimax<HiveMove, HiveState> {
+
+    public getListMoves(node: HiveNode): HiveMove[] {
+        const moves: HiveMove[] = this.getListDrops(node.gameState)
+            .concat(HiveRules.get().getPossibleMovesOnBoard(node.gameState).toList());
```

Comment:
const moves: HiveMove[] = this.getListDrops(node.gameState)
            .concat(HiveRules.get().getPossibleMovesOnBoard(node.gameState).toList());

const moves: HiveMove[] = this.getListDrops(node.gameState)
            .concat(this.getListOfOnBoardMoves(node.gameState)); // Pour l'élégance :p ? j'pourrais même être chaud de le faire en trois ligne :D !

Reply:
refactoré en trois lignes !